contracts/SafeL2.sol

```diff
@@ -2,11 +2,12 @@
 pragma solidity >=0.7.0 <0.9.0;
 
 import {Safe, Enum} from "./Safe.sol";
+import {ISafe} from "./interfaces/ISafe.sol";
```

Comment:
The linter is complaining here - looks like I was wrong about `// solhint-disable-next-line no-unused-import` not being needed.

Reply:
https://github.com/safe-global/safe-smart-account/pull/772/commits/9735a16e8e82abd93710a63c0076ed8b158065fc